docs/configuration/optional-settings.md

```diff
@@ -109,6 +109,16 @@ In order to send email, NetBox needs an email server configured. The following i
 * TIMEOUT - Amount of time to wait for a connection (seconds)
 * FROM_EMAIL - Sender address for emails sent by NetBox
 
+Email is sent from NetBox only for critical events. If you would like to test the email server configuration please use the following:
```

Comment:
We probably want to include a link to Django's documentation for the [`send_mail()` utility](https://docs.djangoproject.com/en/stable/topics/email/#send-mail) in case the reader wants to explore all its options.

Reply:
Included a link to send_mail() documentation